docs/source/basic/library.rst

```diff
@@ -1,16 +1,65 @@
-.. highlight:: cpp
+Library Interface
+=================
+
+As described in the chapter about the :doc:`Abstraction </basic/abstraction>`, the general design of the library is very similar to *CUDA* and *OpenCL* but extends both by some points, while not requiring any language extensions.
```

Comment:
General question: Why are we mentioning OpenCL so often? Except for the experimental SYCL backend we have no relationship to OpenCL.

Reply:
I think because it was like that at the Master thesis used as a basis for these texts

Reply:
Because it was 2015 :sunglasses: . Most parts were copied from Benjamin's master thesis. In my opinion, with overworking the text, we should remove a lot of the OpenCL stuff.